lib/widgets/users/add_first_user_widget.dart

```diff
@@ -59,9 +62,11 @@ class AddFirstUserWidget extends StatelessWidget {
                       Container(
                         width: MediaQuery.of(context).size.width * 0.6,
                         child: TextField(
+                          style: Theme.of(context).textTheme.body1,
                           autofocus: true,
                           textCapitalization: TextCapitalization.words,
                           decoration: InputDecoration(
+                            hintStyle: Theme.of(context).textTheme.body1,
```

Comment:
body1 and body2 seem inconsistent with the naming style you have been using? When is each used? Can we be more specific instead of numbering?

Reply:
body1 and body2 are set by the textTheme and are not changeable, currently the only difference is the color of the two. body1 is used on light backgrounds and body2 is used on dark